pallets/connectors/src/lib.rs

```diff
@@ -123,10 +103,13 @@ pub mod pallet {
 
 	#[pallet::config]
 	pub trait Config: frame_system::Config + pallet_xcm_transactor::Config {
+		/// The overarching event type.
 		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
 
+		/// Weight information for extrinsics in this pallet.
 		type WeightInfo: WeightInfo;
 
+		/// The source of truth for the balance of accounts in native currency.
```

Comment:
We do not need the balance type I think. We could just refer always to the `BalanceOf<T> = <<T as Config>::Tokens as Inspect>::Balance`

Reply:
That does not work as the `Inspect` trait requires setting the associated `Balance` type: https://github.com/paritytech/substrate/blob/ecf71223e6d34204418616d935faad1c35ea36d5/frame/support/src/traits/tokens/fungibles/regular.rs#L51

So we would end up in an evaluation overflow.